test/performance/3-multiple-blobs-tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+echo 'azmi - build executable'
+dotnet build src/azmi-commandline/azmi-commandline.csproj
+
+exePath=$(cd ./src/azmi-commandline/bin/Debug/netcoreapp3.0 || exit; pwd)
+PATH="$PATH:$exePath"
+
+echo "azmi getblobs - performance testing, repeat count: 1"
+echo "VM size: $(curl -sS -H Metadata:true "http://169.254.169.254/metadata/instance/compute/vmSize?api-version=2018-10-01&format=text")"
+echo "azmi: $(azmi --version)"
+CONT="https://azmitest5.blob.core.windows.net/azmi-pt"
+
+
+
+printf  "\n=================\n"
+echo "azmi listblobs -c $CONT"
+time azmi listblobs -c $CONT | wc -l
+sleep 1
```

Comment:
I think this sleep() operation is not required. Commands are executed sequentially here.

Reply:
Uf, this one was weird. I will add comment to it. Commands are sequential, but it was appearing in pipeline log as they are not. Take a look [here](https://dev.azure.com/iiric/azmi/_build/results?buildId=1362&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=c623b9f5-4e91-5154-177e-19d7203ddd0d&l=44). Adding `sleep 1` fixed it. If you have some other explanation or suggestion, tell me.